pkg/content/content.go

```diff
@@ -442,6 +442,15 @@ func readLines(path string) ([]string, error) {
 
 	var lines []string
 	scanner := bufio.NewScanner(file)
+	fileStat, err := file.Stat()
+	if err != nil {
+		return nil, err
+	}
+	if fileStat.Size() > bufio.MaxScanTokenSize {
```

Comment:
wth ?:D

Reply:
buffer size of the scanner is 64 KB, if file is larger than 64K, you can set larger buffer with **scanner.Buffer**